components/notebook-controller/controllers/notebook_controller.go

```diff
@@ -528,9 +529,10 @@ func (r *NotebookReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 	eventToRequest := handler.ToRequestsFunc(
 		func(a handler.MapObject) []ctrl.Request {
+			event := a.Object.(*v1.Event)
 			return []reconcile.Request{
 				{NamespacedName: types.NamespacedName{
-					Name:      a.Meta.GetName(),
+					Name:      nbNameFromInvolvedObject(&event.InvolvedObject),
```

Comment:
I am not sure about this change. Later, in the reconcile call, we need to fetch the actual event:

```
getEventErr = r.Get(ctx, req.NamespacedName, event)
```